src/en/mobile/ios/checklist.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Must-haves criteria for iOS"
+abstract: "Our essential criteria to be followed for iOS applications"
+displayToc: true
+leftColumn:
+  display: false
+  components: 
+    - "wcag-tests-filters"
+js:
+  - /assets/test-ios.js
+---
+
+# iOS mobile application - List of Essential Accessibility Criteria
+
+This set of criteria, designed for iOS applications, helps to avoid the main accessibility errors.
+Its implementation does not guarantee an accessible mobile application, but lays the foundations for a progressive approach to accessibility.
+
+Compliance with the Orange graphic charter called [Design System Orange](https://design.orange.com/) is a prerequisite for using this base.
+Certain points already present in the Orange charter (use of colors, text alignment, etc.) have not been included in this list of criteria.
+
+This guided test grid is intended for designers, developers, qualifiers, product owners, etc. It should be used with the recommended tools to ensure reliable results.
+
+
+<section id="refTests" class="accordion" aria-multiselectable="true">
+  <h2 id="test-textual-content">Textual content</h2>
+  <h2 id="test-images">Images</h2>
+  <h2 id="test-colors-and-contrasts">Colors and contrasts</h2>
+  <h2 id="test-keyboard-navigation">Keyboard navigation</h2>
+  <h2 id="test-text-size">Text size</h2>
+  <h2 id="test-forms">Forms</h2>
+  <h2 id="test-touchsreen-and-interactions">Touchscreen and interactions</h2>  
```

Comment:
~~~suggestion
  <h2 id="test-touchscreen-and-interactions">Touchscreen and interactions</h2> 
~~~